include/fmt/chrono.h

```diff
@@ -769,18 +769,37 @@ template <typename Char, typename Rep, typename OutputIt,
           FMT_ENABLE_IF(std::is_integral<Rep>::value)>
 OutputIt format_duration_value(OutputIt out, Rep val, int) {
   static FMT_CONSTEXPR_DECL const Char format[] = {'{', '}', 0};
-  return format_to(out, compile_string_to_view(format), val);
+
+// Workaround a compiler error in MSVC < 16.8.2
+#if defined(FMT_MSC_VER) && FMT_MSC_VER <= 1928
```

Comment:
My last fix was actually in error and did not hold up to CI once a test was in place